### ISSUE TITLE
V1.1.21: queue corruption?

### ISSUE THREAD

**Issue by erkr** [music-assistant/hass-music-assistant#281]
Added some tracks from a local media playlist to the queue using the add to queue action. They where not added!
Then I tried to start one immediately with play now. That failed with this error:
```

Logger: homeassistant.components.websocket_api.http.connection
Source: custom_components/mass/websockets.py:979 
Integration: Home Assistant WebSocket API (documentation, issues) 
First occurred: 15:17:54 (2 occurrences) 
Last logged: 15:18:02

[547658280528] Error handling message: Unknown error (unknown_error)
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/decorators.py", line 27, in _handle_async_response
    await func(hass, connection, msg)
  File "/config/custom_components/mass/websockets.py", line 96, in async_get_mass_func
    await orig_func(hass, connection, msg, mass)
  File "/config/custom_components/mass/websockets.py", line 979, in websocket_play_media
    await player_queue.play_media(msg[URI], msg[COMMAND])
  File "/usr/local/lib/python3.9/site-packages/music_assistant/models/player_queue.py", line 237, in play_media
    await self.append(queue_items)
  File "/usr/local/lib/python3.9/site-packages/music_assistant/models/player_queue.py", line 415, in append
    next_items = self.items[self._current_index + 1 :] + queue_items
TypeError: unsupported operand type(s) for +: 'NoneType' and 'int'
```

**Comment by marcelveldt** [comment]
released patch release 1.1.22 just now with a fix for this

**Comment by erkr** [comment]
Double issue
